netcore/src/OPADotNet.AspNetCore/OPADotNet.AspNetCore.csproj

```
@@ -1,31 +1,31 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <PackageLicenseFile>LICENSE</PackageLicenseFile>
-    <RepositoryUrl>https://github.com/koralium/OPADotNet</RepositoryUrl>
-    <PackageProjectUrl>https://github.com/koralium/OPADotNet</PackageProjectUrl>
-  </PropertyGroup>
+	<PropertyGroup>
+		<TargetFramework>net6.0</TargetFramework>
```

```suggestion
		<TargetFrameworks>netcoreapp3.1;net6.0</TargetFrameworks>
```

Since we need to be backwards compatible